warrior/Framework/Utils/testrandom_utils.py

```diff
@@ -0,0 +1,314 @@
+                    and testcase == ?", (exec_tag, testcase_name, ))
+        else:
+            file_name = "{}_report.xlsx".format(exec_tag)
+            records = con.execute("SELECT records from GEN_RESULTS_TABLE where tag == ?", (exec_tag, ))
+    for record in records:
+        all_samples.extend(json.loads(record[0]))
+    writer = pd.ExcelWriter("{}".format(file_name))
+    df_db = pd.DataFrame(all_samples)
+    df_summary = pd.DataFrame([{"Testcase" : os.path.basename(testcase_name),
+                                "Total Samples": no_of_combinations,
+                                "Total Samples Executed":len(df_db.index),
+                                "Passed" :(df_db.result == 'PASS').sum(),
+                                "Failed" : (df_db.result == 'FAIL').sum()}])
+    df_summary.to_excel(writer, index=False, sheet_name='summary')
+    df_db.to_excel(writer, index=False, sheet_name=os.path.basename(testcase_name))
+    writer.save()
+    writer.close()
+    return file_name
+
+def get_samples_shuffle_columns(df, no_of_samples, records_in_db):
+    """ get samples for shuffle columns"""
+    generic_data_dict = []
+    all_dict = []
+    df_dict = {}
+    supported_combinations = 100000
+    no_of_combinations = 1
+    for column in df.columns:
+        df_dict[column] = [x for x in df[column].sample(frac=1) if not pd.isnull(x)]
+        no_of_combinations *= len(df_dict[column])
+    if no_of_combinations > supported_combinations:
+        print_error("Total no. of random combinations found in given xls file : {}, "
+                    "max supported combinations with --select_random_values option set is {}"
+                    ", please reduce column size in given variable data xls file"\
+                            .format(no_of_combinations, supported_combinations))
+        exit(1)
+    column_names = df_dict.keys()
+    column_values = df_dict.values()
+    for tup in product(*column_values):
+        dict1 = {}
+        i = 0
+        for column_name in column_names:
+            dict1[column_name] = tup[i]
+            i+=1
+        all_dict.append(dict1)
+    print_info("================================================================================")
+    print_info("Total no. of combinations found in variable xls file : {}".format(len(all_dict)))
+    print_info("Total no. of combinations already tested : {}".format(len(records_in_db)))
+    print_info("Total no. of combinations remaining to test : {}".format(len(all_dict) - len(records_in_db)))
+    print_info("Total no. of random combinations selected in this test : {}".format(no_of_samples))
+    print_info("================================================================================")
+    #get samples not in db.
+    for db_sample in records_in_db:
+        if db_sample in all_dict:
+            all_dict.remove(db_sample)
+    if len(all_dict) == 0:
+        print_error("All the samples are tested. use --reset_results to restart test again")
```

Comment:
The error message using the old cli  option.  The option should be --reset_execution